tests/cypress/tests/e2e/admin-dashboard/admin-dashboard.stepdef.js

```diff
@@ -0,0 +1,25 @@
+import { When, Then } from "@badeball/cypress-cucumber-preprocessor";
+
+When("I select a state", () => {
```

Comment:
**Request**

The state Abbreviation should be passed in as parameter, so the same step could be run with 1,2, or all 50 states.